woocommerce/woocommerce-ios
# Add-Ons: Add beta indicator banner

Part of #3894  

# Why

This PR does 3 things(Sorry, I know it should do only one thing!)

1. Update the info label below the add-on list to have a new copy and to have a bit more padding.

2. Add a background color/view to matches the design better

3. Show a beta indicator top banner. Button actions will be handled later

# How
1. Just changed the copy in the localization menu and added some top padding to `OrderAddOnNoticeView`

2. Update `OrderAddOnListI1View` to have a solid color as a background view, so it can expand beyond the safe areas and cover the whole screen.

3. To read the whole story: paNNhX-dd-p2

	- Created `TopBannerWrapperView` in order to have a view that wraps a `TopBannerView` but allows us to have a custom `intrinsicContentSize` without having to modify the legacy of `TopBannerView`. This is needed because `SwiftUI` is unable to render a `TopBannerView` instance due to how its constraints are built. 
	
	- Created a `OrderAddOnTopBanner` which is a `struct` that wraps a `UIKitView` using the `UIViewRepresentable` protocol, to allow `SwiftUI` to represent it. The `UIKit` view that this wraps is a `TopBannerWrapperView` already configured to show the "beta banner" content as well as provide functions to get notified when the action buttons are pressed.
	
	- Update `OrderAddOnListI1View` to include the new `OrderAddOnTopBanner` created in the step above.
	
# Demo

![demo](https://user-images.githubusercontent.com/562080/115775206-32494e00-a378-11eb-9309-78b601d919e6.gif)

# Testing steps

**Prerequisites:** Have your site with the add-ons plugin installed and with at least one order with add-ons.

- Launch the app and navigate to the add-on order
- Tap on the "View Add-Ons" button.
- See that there is a "beta feature" banner
- Tap on the chevron button and make sure the banner collapses/expands accordingly
 
Update release notes:
